src/ucp/proto/proto_debug.c

```diff
@@ -512,9 +512,13 @@ ucp_proto_perf_node_new(ucp_proto_perf_node_type_t type, const char *name,
     }
 
     perf_node->type     = type;
-    perf_node->name     = name;
     perf_node->refcount = 1;
     ucs_array_init_dynamic(&perf_node->children);
+
+    if (name == NULL) {
```

Comment:
in which flow the name can be NULL?

Reply:
Changed to assert

Reply:
Answering you question: `rndv/rkey_ptr` proto set `params.send_op` to `UCT_EP_OP_LAST` which causes setting `uct_ep_operation_names[UCT_EP_OP_LAST]` which is NULL to name of the `ucp_proto_common_lane_perf_node` result node.